lxl-web/src/routes/(app)/[[lang=lang]]/(find)/Pagination.svelte

```diff
@@ -57,16 +64,24 @@
 			<!-- last and next -->
 			{#if !isLastPage}
 				{#if sequenceEnd !== lastPage}
-					<li><a href={relativizeUrl(last['@id'])}>{lastPage}</a></li>
+					<li><a class="button-ghost" href={relativizeUrl(last['@id'])}>{lastPage}</a></li>
 				{/if}
-				<li><a href={relativizeUrl(next?.['@id'])} aria-label="Nästa sida">→</a></li>
+				<li>
+					<a class="button-ghost" href={relativizeUrl(next?.['@id'])} aria-label="Nästa sida"
+						><BiChevronRight class="text-icon"></BiChevronRight></a
+					>
+				</li>
 			{/if}
 		</ul>
 	</nav>
 {/if}
 
 <style>
```

Comment:
Adding `<style lang="postcss">`should remove the `Unknown at rule @apply` warning.